src/Core/src/Fonts/EmbeddedFont.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.IO;
+
+namespace Microsoft.Maui
+{
+	public class EmbeddedFont
```

Comment:
This is a bad name as this is passed to an "OS font installer" concept. This is really a filename to use when saving this stream to disk.